riot/riot.py

```diff
@@ -109,6 +118,17 @@ def venv_path(self) -> str:
         version = self.version().replace(".", "")
         return f".riot/venv_py{version}"
 
+    def site_packages_path(self) -> str:
```

Comment:
This function feels a little brittle. Is there a way to get the site packages path from the interpreter itself?

A quick google suggests that maybe `python -c 'import site; print(site.getsitepackages())'` would do the trick

Reply:
Cool, I do actually like that.

Do you think we should grab all of the site packages directories? Or should it always be 1 element and we can get away with `site.getsitepackages()[0]`?

Reply:
I suppose the function could return all since it might be useful in the future.